modules/installation-openstack-ovs-dpdk-performance-profile.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+// TODO: ^^^ at the moment
+
+:_content-type: REFERENCE
+[id="installation-openstack-ovs-dpdk-performance-profile_{context}"]
+= A performance profile template for clusters that use OVS-DPDK on OpenStack
+
+To maximize machine performance in a cluster that uses Open vSwitch with the Data Plane Development Kit (OVS-DPDK) on {rh-openstack-first}, you can use a performance profile.
+
+You can use the following performance profile template to create a profile for your deployment.
+
+.A performance profile template for clusters that use OVS-DPDK
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: cnf-performanceprofile
+spec:
+  additionalKernelArgs:
+    - nmi_watchdog=0
+    - audit=0
+    - mce=off
+    - processor.max_cstate=1
+    - idle=poll
+    - intel_idle.max_cstate=0
+    - default_hugepagesz=1GB
+    - hugepagesz=1G
+    - intel_iommu=on
+  cpu:
+    isolated: <CPU_ISOLATED>
+    reserved: <CPU_RESERVED>
+  hugepages:
+    defaultHugepagesSize: 1G
+    pages:
+      - count: <HUGEPAGES_COUNT>
+        node: 0
+        size: 1G
+  nodeSelector:
+    node-role.kubernetes.io/worker: ''
+  realTimeKernel:
+    enabled: false
+    globallyDisableIrqLoadBalancing: true
+----
+
+Insert values that are appropriate for your configuration for the `CPU_ISOLATED`, `CPU_RESERVED`, and `HUGEPAGES_COUNT` keys. 
```

Comment:
Will your target audience know what's appropriate? If not, consider adding annotations (that use the where: DL format) to explain these keys.

Reply:
Yeah, I'd expect the audience to know these at this point. Anyone doing this is going to be intimately familiar with Scalability and performance.